saas/fsm/user_status_change/fsm-user_status_change.pt

```diff
@@ -0,0 +1,250 @@
+name "SaaS Manager - User Status Change"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will raise an alert when Flexera SaaS Manager identifies users whose status in the HR roster changes to inactive. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/user_status_change/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
product doesn't want to use alert. instead change **raise an alert** to **create an incident.**